e2e/journey-app/server-configs.ts

```diff
@@ -9,8 +9,8 @@ import type { JourneyClientConfig } from '@forgerock/journey-client/types';
 export const serverConfigs: Record<string, JourneyClientConfig> = {
   basic: {
     serverConfig: {
-      baseUrl: 'https://openam-sdks.forgeblocks.com/am/',
+      baseUrl: 'http://localhost:9443/am',
     },
-    realmPath: '/alpha',
+    realmPath: 'root',
   },
```

Comment:
Should we have a different config object for the Mock API, rather than replacing our shared test tenant?